src/cloud-providers/ibmcloud/provider.go

```diff
@@ -177,6 +177,13 @@ func (p *ibmcloudVPCProvider) getInstancePrototype(instanceName, userData, insta
 			Enabled:  core.BoolPtr(true),
 			Protocol: core.StringPtr(vpcv1.InstanceMetadataServicePatchProtocolHTTPConst),
 		},
+		ConfidentialComputeMode: core.StringPtr(vpcv1.InstanceConfidentialComputeModeDisabledConst),
+		EnableSecureBoot:        core.BoolPtr(false),
+	}
+
+	if p.serviceConfig.ConfidentialComputing {
```

Comment:
Why not use the existing option of disablecvm used with existing providers ?

Reply:
Good point! I missed that so just made up a property myself. I will change it.

Reply:
Removed this part and will do it properly in a followup PR. This PR is now just updating the dependency.